src/Analyzers/CSharp/Tests/UseCollectionExpression/UseCollectionExpressionForEmptyTests.cs

```diff
@@ -1633,6 +1633,27 @@ void M()
         }.RunAsync();
     }
 
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70996")]
+    public async Task TestInterfaceOn_ReadWriteDestination()
+    {
+        await new VerifyCS.Test
+        {
+            TestCode = """
+                using System;
+                using System.Collections.Generic;
+
+                class C
+                {
+                    void M()
+                    {
+                        IList<int> v = Array.Empty<int>();
```

Comment:
💡 Also add a test (likely `[Theory]`) for the target collection types `IReadOnlyCollection<T>` (allowed), `IReadOnlyList<T>` (allowed), and `ICollection<T>` (disallowed)

Reply:
good idea.